packages/core/index.d.ts

```diff
@@ -433,6 +433,14 @@ declare module '@rjsf/core' {
         ): number;
 
         export function schemaRequiresTrueValue(schema: JSONSchema7): boolean;
+
+        export function descriptionId(id: string): string;
```

Comment:
Let's add one-line comments before each new function explaining what they do.

Reply:
Sure! I commented `packages/core/src/utils.js` as most of the comments are in that file.